src/core/2d_primitives.js

```diff
@@ -390,12 +390,16 @@ p5.prototype.quad = function() {
  * @param  {Integer} [detailY] number of segments in the y-direction
  * @chainable
  */
-p5.prototype.rect = function(x, y, w, h, detailX, detailY) {
+p5.prototype.rect = function(x, y, w, h) {
```

Comment:
this signature initially confused me because it looked like detailX and detailY were removed completely.
would it make more sense to leave the parens empty here instead?